03/app.js

```diff
@@ -1,2 +1,18 @@
 const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const oddNumbers = [];
+
+function pushNumber() {
+
+    while(oddNumbers.length < n) {
+
+        const randomNumber = Math.round( Math.random() * 100 );
+
+        if ( !(randomNumber%2 === 0) ) {
+            oddNumbers.push(randomNumber)
+        }
+    }
+    console.log('The drawn numbers are' , oddNumbers);
+}
+
+pushNumber();
```

Comment:
W tym zadaniu chodziło o to, aby pobrać liczby nieparzyste dla danego ciągu znaków 
np. jeśli n = 4 to nieparzystymi są: [1, 3]. Nie ma to być zupełna losować jak to przedstawiłeś tutaj :)

Reply:
`
function pushNumber() {

 for(let i=0; i<=n; i++) {

    if( !(i%2 === 0) ) {
        oddNumbers.push(i)
    }
 }
}

console.log('Yours numbers are: ' , oddNumbers);
pushNumber();`